compiler-rt/lib/builtins/cpu_model/aarch64/fmv/windows.inc

```diff
@@ -0,0 +1,42 @@
+#ifndef _ARM64_
+#define _ARM64_
```

Comment:
Why this define here?

Reply:
winnt.h complains about it if not defined and clang doesn't define it.

Reply:
That's interesting - I would generally expect it to work to include those headers as is, with the compiler default defines.

Can you boil that down to a standalone repro, with a file that just includes `<windows.h>` or similar (I guess the relevant one here is `<processthreadsapi.h>`)? Or is the case that `<processthreadsapi.h>` is one of the Windows headers that you can't include directly unless you've included `<windows.h>` first? In that case, I'd prefer including that first, rather than adding our own extra defines. (If you want to, you can define `WIN32_LEAN_AND_MEAN` to reduce the overhead of including the whole `<windows.h>`.)

Reply:
`windows.h` defines it so let's do that.